### DISCUSSION TITLE
When stacked overfitting occurred: True

### DISCUSSION THREAD

**Discussion by giobrus** [autogluon/autogluon#3796] [Q&A] [answered]
Hi everyone, I have a question: what can i do if when after calling:

`predictor = TabularPredictor(label='num', eval_metric='accuracy',path='run_5').fit(train_df, time_limit=86400, presets='best_quality',num_bag_folds=8,num_bag_sets=20,num_stack_levels=3,dynamic_stacking=True)`

I get **Stacked overfitting occurred: True**?

Is it a problem of train/test split? (currently 0.8/0.2)? or is the dataset way too small? (303 instances) or should I change parameters? I have tried without bagging parameters, with ds_args parameters (ds_args={'validation_procedure':'cv','n_folds':10,'n_repeats':2}) and with none of them, but I still get stacked overfitting occurred: True

Thank you

**Comment by Innixma** [comment, marked as answer]
Refer to #2779. It is not a problem with your approach, but rather stacking is not effective for the problem so it is disabling it dynamically to improve the final model quality.
